arduino/driverStation.py

```
@@ -0,0 +1,77 @@
+    print("------------------------------------------------", flush=True)
+    print(" Jetson PLC Running: 6 Buttons Mode", flush=True)
+    print(" Pins: 29, 31, 32, 33, 35, 37", flush=True)
+    print("------------------------------------------------", flush=True)
+    
+    t = threading.Thread(target=background_loop)
+    t.daemon = True
+    t.start()
+    
+    StartTcpServer(context, address=("0.0.0.0", 502))
+    GPIO.cleanup()
+EOF
```

Ruff (0.15.1)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)